core/graph/impl/src/main/kotlin/bindings/mixins.kt

```diff
@@ -93,9 +93,13 @@ internal interface BindingDefaultsMixin : Binding, BaseBindingDefaultsMixin {
             }
         }
 
-        if (scopes.isNotEmpty() && scopes notIntersects owner.scopes) {
+        if (!owner.canHost(scopes)) {
             validator.reportError(Strings.Errors.noMatchingScopeForBinding(binding = this, scopes = scopes))
         }
+
+        if (ScopeModel.Reusable in scopes && scopes.size > 1) {
+            validator.reportWarning(Strings.Warnings.reusableScopeShouldBeSingle())
```

Comment:
This should be error because other scopes lose sense when adding Reusable which can be unexpected

Reply:
Fixed.